app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,cat_dog_human_age",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
+        ],
+        ids=[
+            "0 cat/dog years should convert into 0 human age.",
+            "14 cat/dog years should convert into 0 human age.",
+            "15 cat/dog years should convert into 1 human age.",
+            "23 cat/dog years should convert into 1 human age.",
+            "24 cat/dog years should convert into 2 human age.",
+            "27 cat/dog years should convert into 2 human age.",
+            "28 cat/dog years should convert into 3/2 human age.",
+            "100/100 cat/dog years should convert into 21/17 human age.",
+        ]
+    )
```

Comment:
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/93856423/779f8582-7994-4789-ae10-b264d401c37d)